doc/lua_api.txt

```diff
@@ -1573,10 +1596,12 @@ If the result digging time is 0, a delay of 0.15 seconds is added between
 digging nodes; If the player releases LMB after digging, this delay is set to 0,
 i.e. players can more quickly click the nodes away instead of holding LMB.
 
-#### Damage groups
+Damage groups
```

Comment:
This is actually `###` a level below `Tools definition`as it describes a part of the tool definition.